README.md

```diff
@@ -7,6 +7,35 @@ jitsi-meet-electron-utils contains native code for some utilities. You'll need [
 NOTE: For Linux install libxtst-dev and libpng++-dev (`sudo apt-get install libxtst-dev libpng++-dev`). This dependancies are related to RobotJS which is a dependency of jitsi-meet-electron-utils. You can see the build instructions for RobotJS [here](https://github.com/jitsi/robotjs/tree/jitsi#building)
 
 ## Usage
+#### Google API
+The Google API utility exposes methods for performing Google client-side authentication by opening a oauth popup window and passing back the redirect URL with the Google access token.
+
+**Enable the Google API integration:**
+
+In the **main** electron process:
+```Javascript
+const { setupGoogleApiMain } = require("jitsi-meet-electron-utils");
+const options = {
+    browserWindowOptions: {
```

Comment:
Aren't this the options by default? What's the point of passing them?

Reply:
Those are the default options. I just wanted to pass an example of passing browserWindowOptions. Do you think that's unclear for a doc? Should I use pseudo code instead to make clear that electron browser window options can be passed?